packages/generator/src/service/package-json.ts

```diff
@@ -1,15 +1,14 @@
 export function packageJson(
   npmPackageName: string,
-  versionInPackageJson: string | undefined,
-  generatorVersion: string,
+  version: string,
```

Comment:
The idea of two parameters is you can generate a client with `1.0.0` version but uses dependency `sdk-core` version `2.0.0`.

Reply:
oh, I misssed that thanks for finding the mistake.